datasets/cta/perf_prod2/README

```diff
@@ -0,0 +1,10 @@
+This repository contains the official CTA's instrument response functions (IRF, https://portal.cta-observatory.org/CTA_Observatory/performance/SitePages/Home.aspx). You can find in FITS and ROOT format in:
```

Comment:
Please rename `datasets/cta/perf_prod2/README` to `datasets/cta/perf_prod2/README.md` (or `.rst`, whichever format you prefer).

Add a title line
```
# CTA prod2 performance files
```
and break some of the other very long lines, so that they are easier to read e.g. here on Github in the diff.